tools/build_framework.sh

```diff
@@ -94,12 +96,19 @@ if [ x"$cli_only" = xtrue ]; then
     exit
 fi
 
-if [ -n "$BOOTSTRAP_DIR" ]; then
+DISABLED_VALUE="disable"
+
+if [ "$BOOTSTRAP_DIR" != $DISABLED_VALUE ]; then
```

Comment:
I know this is merged, but just wanted to point out that these conditional blocks here are redundant now as the output for both branches will be the same if the value is set to `"disabled"`.